docs/content/docs/bundlers.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fluid and Bundlers
+aliases:
+  - "/start/bundlers/"
+---
+
+The following are known bundler issues, answers to some of the most commonly asked questions about the Fluid
+Framework and Bundlers.
+
+## Known Errors with Webpack 5
+We know the following packages cause errors; assert, buffer, events and url.
```

Comment:
It's not clear what these 4 things are? Are they part of FF? Are they part of WebPack? Are they popular JS libraries?

Reply:
What about calling them core NodeJS Modules?